modules/tinymce/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Resizing table columns in some scenarios would resize the column to an incorrect position #TINY-7731
 - Image resize backdrop element did not have `data-mce-bogus="all"` set #TINY-7854
 - Clicking beside a non-editable element could cause the editor to incorrectly scroll to the top of the document #TINY-7062
+- Disabled `window.visualViewport` in Mozilla Firefox which caused a bug preventing the sticky toolbar from appearing in fullscreen mode #TINY-7873
```

Comment:
The first part of this should be in the Sugar changelog, not here. This changelog should be solely about fixing the toolbar/header disappearing.

- [ ] Fix changelog and ensure we have a sugar changelog as well.